src/main/java/codeu/model/data/Hashtag.java

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package codeu.model.data;
+
+import java.time.Instant;
+import java.util.UUID;
+
+/**
+ * Class representing a hashtag
+ */
+public class Hashtag {
+  public final UUID id;
+  public final UUID poster;
+  public final Instant creation;
+  public final String hashTitle;
+
+  /**
+   * Constructs a new Hashtag.
+   *
+   * @param id the ID of this Hashtag
+   * @param owner the ID of the User who created this Hashtag
+   * @param title the title of this Hashtag
+   * @param creation the creation time of this Hashtag
+   */
+  public Hashtag(UUID id, UUID poster, String hashTitle, Instant creation) {
```

Comment:
how do you imagine we look up which conversations/messages contain hashtags? should we have a UUID for 'poster' and a map/list of UUID to type [MESSAGE, CONVERSATION] that this hashtag is used for?

Reply:
yes

Reply:
done changes